docs/proto/proto-docs.md

```diff
@@ -238,6 +257,21 @@ Query provides defines the gRPC querier service
 
 
 
+<a name="osmosis.meshsecurity.v1beta1.ScheduledWork"></a>
+
+### ScheduledWork
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| `repeat` | [bool](#bool) |  |  |
```

Comment:
Description for this field / section would be nice. What is `ScheduledWork`? What is this `repeat` field?